src/dosbox.cpp

```diff
@@ -405,7 +405,7 @@ void DOSBOX_Init(void) {
 	const char *iosgus[] = { "240", "220", "260", "280", "2a0", "2c0", "2e0", "300", 0 };
 	const char *irqsgus[] = { "5", "3", "7", "9", "10", "11", "12", 0 };
 	const char *dmasgus[] = { "3", "0", "1", "5", "6", "7", 0 };
-
+	const char *lfn_settings[] = { "true", "false", "auto", "autostart", 0};
```

Comment:
True and false are usually used for boolean conf types; where as this is string variable.
suggest using "yes/no" or "on / off", and then picking auto or autostart, and not giving multiple options that mean the same thing. 

(longer term though, the dosbox ini needs to be simplified and the true/false, yes/no, on/off stuff boiled down to one pair of names so users aren't confused.  I'm still caught by this alot..)

Reply:
Done. I have already renamed "true" and "false" to "on" and "off" respectively for this.